xbmc/addons/interfaces/GUI/AddonCallbacksGUI.cpp

```diff
@@ -1494,10 +1494,8 @@ GUIHANDLE CAddonCallbacksGUI::ListItem_Create(void *addonData, const char *label
     pItem->SetLabel(label);
   if (label2)
     pItem->SetLabel2(label2);
-  if (iconImage)
-    pItem->SetIconImage(iconImage);
-  if (thumbnailImage)
-    pItem->SetArt("thumb", thumbnailImage);
+  if (iconImage || thumbnailImage)
+    CLog::Log(LOGWARNING, "Usage of iconImage or thumbnailImage results in NOP and will soon be removed. Use setArt instead.");
```

Comment:
Add a:
```cpp
  if (iconImage)
    pItem->SetArt("icon", thumbnailImage);
  if (thumbnailImage)
    pItem->SetArt("thumb", thumbnailImage);
```
This class to handle `libKODI_guilib.h` should be removed in furtue (if PVR and Game addons are changed to new).
For the class constructor there, I would therefore leave the support first.

Reply:
Thanks. I just want to make it NOP for python so, reverted the change